data_utils/comment_classification.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Module for code review comment classification. """
+
+import csv
+import os
+import random
+import sys
+
+
+def main():
+    """" Creates training dataset with 200 host code review comments.
+
+    The 200 comments from hosts are selected at random.
+
+    Args:
+        comments_file: CSV containing all code review comments.
+    """
+
+    if len(sys.argv) == 1:
+        comments_file = "data/pr_comments.csv"
+        training_file = "data/training_comments.csv"
+    else:
+        if sys.argv[1] == "test":
+            comments_file = "data/test_pr_comments.csv"
+            training_file = "data/test_training_comments.csv"
+        else:
+            raise Exception("Invalid command line argument.")
+
+    if not os.path.isfile(comments_file):
+        raise Exception("The CSV for code review comments does not exist.")
+
+    random.seed(10)
```

Comment:
do you want it to be deterministic? If so, add a comment.

Reply:
Yes, because I wanted to be able to generate the same comments in case I had to re-run after fixing any bugs. I'll add a comment.

Reply:
okay -- do you want that to be nondeterministic in the future, or are you leaving that in for production

Reply:
Hm, I was thinking that if I were to do this again, I'd probably label different comments. Should I delete that part?

Reply:
I'd comment it out and flip the comment to suggest that you should uncomment that line if you want deterministic behavior.

Reply:
Okay, I'll do that then!